crosswalks/ogcapi-coverages.md

```diff
@@ -0,0 +1,428 @@
+`properties` parameter and [load_collection](https://processes.openeo.org/#load_collection)
+process respectively. Both band names and band indices can be used in **OAC1**, 
+while **openEO** accepts band names exclusively.
+
+An **openEO** server can optionally provide a rough estimate of the time/cost
+of a stored job via [`/jobs/{jobId}/estimate`](https://api.openeo.org/1.2.0/#tag/Data-Processing/operation/estimate-job).
+Server logs are also available in **openEO** either directly in the synchronous processing
+response (as a link), or via the API resources dedicated to the batch jobs
+(eg. [`jobs/{jobId}/logs`](https://api.openeo.org/1.2.0/#tag/Batch-Jobs/operation/debug-job)).
+
+
+### Data scaling
+
+- openEO:
+  - [`POST /result`](https://api.openeo.org/1.2.0/#tag/Data-Processing/operation/compute-result) @
+[`resample_spatial`](https://processes.openeo.org/#resample_spatial) process (sync)
+  -  [`POST /jobs`](https://api.openeo.org/1.2.0/#tag/Batch-Jobs/operation/create-job) @
+[`resample_spatial`](https://processes.openeo.org/#resample_spatial) process +
+[`POST /jobs/{jobId}/results`](https://api.openeo.org/1.2.0/#tag/Batch-Jobs/operation/start-job) (async)
+
+- OAC1: 
+  - `GET /collections/{collectionId}/coverage?scale-size=axisName({number})[,axisName({number})]*`
+  - `GET /collections/{collectionId}/coverage?scale-axes=axisName({number})[,axisName({number})]*`
+  - `GET /collections/{collectionId}/coverage?scale-factor={number}`
+
+Rescaling a dataset in **openEO** requires the same procedure as for any other kind
+of processing, thus preparing a process graph that executes the desired operation, then
+fetch the results, either synchronously or asynchronously through the creation of a
+[batch job](https://api.openeo.org/1.2.0/#tag/Data-Processing/operation/create-job).
+
+At the time of writing, the available **openEO** processes for data resampling are
+[`resample_spatial`](https://processes.openeo.org/#resample_spatial) &mdash; 
+that rescales the spatial dimensions to a target resolution, or to
+a target spatial coordinate reference system &mdash; and
+[`resample_cube_spatial`](https://processes.openeo.org/#resample_spatial)/
+[`resample_cube_temporal`](https://processes.openeo.org/#resample_cube_temporal), that
+rescale the spatial/temporal dimensions to match the resolutions of a
+provided target datacube.
+
+While **OAC1** does not let a user specify
+a resolution, it lets a client specify the target number of samples
+along a dimension, or the scaling factor. While the capabilities
+of a user-defined process with asynchronous job management
+go well beyond data the resampling of a dataset and provide 
+a wide range of benefits, **OAC1** provides a somewhat simpler shortcut
+for via query parameters of the URL.
+
+**OAC1** also allows for scaling on *any* coverage's dimension, while
+**openEO** only along the spatial plane/dimensions.
+
+
+### Tiles
+
+- openEO: `n/a`
```

Comment:
openEO allows it through secondary web services, similar to OAC1.

Reply:
Yes, I mention that, but I'll make it clearer by replacing the n/a also.